packages/eslint-config-airbnb/rules/react.js

```diff
@@ -465,6 +466,14 @@ module.exports = {
     // https://github.com/yannickcr/eslint-plugin-react/blob/bc976b837abeab1dffd90ac6168b746a83fc83cc/docs/rules/jsx-fragments.md
     // TODO: enable, semver-major
     'react/jsx-fragments': ['off', 'syntax'],
+
+    // Enforce Rules of Hooks
+    // https://github.com/facebook/react/blob/f99fca3cb28e1fbd87e0b647ac43959add290be7/packages/eslint-plugin-react-hooks/README.md
+    'react-hooks/rules-of-hooks': 'error',
+
+    // Verify the list of the dependencies for Hooks like useEffect and similar
+    // https://github.com/facebook/react/blob/f99fca3cb28e1fbd87e0b647ac43959add290be7/packages/eslint-plugin-react-hooks/README.md
```

Comment:
The official repo doesn't have a separate README file for each rule, so I linked to the main one. There's also this link: https://reactjs.org/docs/hooks-rules.html

Reply:
At the least, each rule should have a permalink - if not, can you file an upstream issue for them to add one?

Reply:
I can get a permalink to the actual rule implementation. Does that work?

There aren't any dedicated READMEs for each one.